### PR TITLE
Unit tests of fetch pre post assessment score api

### PR DESCRIPTION
**Pull request by abdullah998**

### CODE REVIEW

File: openedx/features/philu_courseware/tests/test_competency_assessment_scores_api.py [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    def get_competency_assessment_api_response(self, chapter_id):
+        """Method to create url and call api"""
+        url = reverse(
+            'competency_assessments_score',
+            kwargs={
+                'course_id': self.course.id,
+                'chapter_id': chapter_id,
+            }
+        )
+
```
Comment (aliassad) [inline openedx/features/philu_courseware/tests/test_competency_assessment_scores_api.py]: remove this space

File: openedx/features/philu_courseware/tests/test_competency_assessment_scores_api.py [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+        response = self.get_competency_assessment_api_response(chapter.location.block_id)
+        self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
+
+    def test_competency_assessment_chapter(self):
+        """Test module/chapter has both pre and post assessments"""
+        chapter = self.create_chapter()
+        self.create_assessment_subsection(chapter)
+        self.create_assessment_subsection(chapter, constants.POST_ASSESSMENT_FORMAT)
+        response = self.get_competency_assessment_api_response(chapter.location.block_id)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
```
Comment (aliassad) [inline openedx/features/philu_courseware/tests/test_competency_assessment_scores_api.py]: please check , is there is any way to reduce these assert checks ?

File: openedx/features/philu_courseware/tests/test_competency_assessment_scores_api.py (hunk 1 above)
Comment (abdullah998) [reply openedx/features/philu_courseware/tests/test_competency_assessment_scores_api.py]: Corrected

File: openedx/features/philu_courseware/tests/test_competency_assessment_scores_api.py (hunk 2 above)
Comment (abdullah998) [reply openedx/features/philu_courseware/tests/test_competency_assessment_scores_api.py]: I've checked that there are cases in our unit tests where we use multiple asserts in a single unit test

**Pull request merged**